### PR TITLE
docs(Readme.Md): docs for configurations

### LINKED ISSUE #46: Document test.conf
Create instructions on setting up `test.conf` on `README.md` (Setup test-suite section)

### PR DESCRIPTION
**Pull request by kdhttps**
Added details about configuration in readme.md

closes #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,8 +75,24 @@ ls /root
...
+|-----|-------------|
+|**PASSPORT_HOST**|Use to set Host for Passport Droplet|
+|**PASSPORT_IP**|Use to set floating IP for Passport Droplet|
+|**LATEST_DEV_SNAPSHOT_ID**|It is `Dev Passport Snapshot Id` from which you want to create droplet for Passport|
+|**LATEST_STABLE_SNAPSHOT_ID**|It is `Stable Passport Snapshot Id` from which you want to create droplet for Passport|
+|**PROVIDER_HOST**|Use to set Host for SAML IDP Provider Droplet|
+|**PROVIDER_IP**|Use to set floating IP for SAML IDP Provider Droplet|
+|**PROVIDER_SNAPSHOT_ID**|It is `SAML IDP Provider Snapshot Id` from which you want to create droplet for SAML IDP Provider|
+|**CLIENT_HOST**|It is use to set Host for Request Party Client i.e. [auth-tdd-client](https://github.com/christian-hawk/auth-tdd-client)|
+|**API_CLIENT_ID**|It is OP Client's ClientID which use by `auth-tdd-client` during passport auth flow|
```
Comment (christian-hawk) [inline README.md:94]: Actually here is the **Gluu 4.1.1 Admin API** to automate user creation on **PROVIDER_HOST** server.

**Review by christian-hawk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kdhttps) [reply README.md:94]: Pardon! Not getting you. you mean this API_Client is for automating user create on PROVIDER_HOST using Admin API, right?
If yes then the description for it like "It is OP Client's ClientID which is used to create user on SAML IDP Gluu Server using Gluu 4.1.1 Admin API".
is this ok?

Comment (christian-hawk) [reply README.md:94]: No problem, feel free to ask all questions. Lemme try to explain.
Here we a `gluu-server` instance (4.1.1) call `PROVIDER` (from passport provider), that acts as `IDP` (saml) or `OP`.
This `gluu-server` instance has an **Gluu Admin API** enabled,  test-suite uses `create_user.py`, and some other scripts to call **Gluu Admin API** w/ `API_CLIENT_ID` and `API_CLIENT_SECRET** as credentials.

Comment (kdhttps) [reply README.md:94]: Ok Got it, thank you :) .
So is this below description is ok?
`API_CLIENT_ID: "It is OP Client's ClientID which is used to create user on SAML IDP Gluu Server using Gluu 4.1.1 Admin API"`

Comment (christian-hawk) [reply README.md:94]: not sure thou. What do you think about sth like:

*Providers' gluu-server [Admin REST API](https://gluu.org/docs/gluu-server/4.1/api-guide/oxtrust-api/) `client_id` (used for automated user creation/deletion)*

?

Comment (kdhttps) [reply README.md]: Done, thank you :).

**Review by christian-hawk — APPROVED** [review]

**Pull request merged**